<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalculadoraBasica</class>
 <widget class="QWidget" name="CalculadoraBasica">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QPushButton" name="Cal_Btt_goToIni">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>230</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Ir a Inicio</string>
   </property>
  </widget>
  <widget class="QLabel" name="Cal_lbl_1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>301</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Sans Serif</family>
     <pointsize>22</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>Calculadora Básica</string>
   </property>
  </widget>
  <widget class="QLabel" name="Cal_lbl_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>100</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Número</string>
   </property>
  </widget>
  <widget class="QLabel" name="Cal_lbl_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>59</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>Número</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Cal_ledit_Num1">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Cal_ledit_Num2">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>130</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Cal_ledit_cal">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>190</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Source Code Pro</family>
     <pointsize>12</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(170, 177, 165);</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Cal_comB">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>190</y>
     <width>79</width>
     <height>23</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Suma</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Resta</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>División</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Multiplicación</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="Cal_Btt_calSuma">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>160</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Suma</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cal_Btt_goToIni_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>80</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Limpiar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Cal_Btt_calResta">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>160</y>
     <width>41</width>
     <height>23</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>7</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Resta</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>Cal_Btt_goToIni_3</sender>
   <signal>clicked()</signal>
   <receiver>Cal_ledit_cal</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>202</y>
    </hint>
    <hint type="destinationlabel">
     <x>163</x>
     <y>201</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
